research/04-problem-def.tex

```diff
@@ -12,15 +12,15 @@ \section{Problem Definition}
 convenient addition to the product. \Cref{sec:repl} showed that REPLs
 provide this rapid prototyping ability for other languages like Lisp
 and Python. Therefore, the client has expressed their interest in
-REPLs that are generated from language definitions created with
+a REPL that works with all language definitions created in
 Spoofax. Such a REPL would aid both the developer of the language and
 its end-user.
 
 Since a lot of language services are already provided by Spoofax, the
 REPL should try to hook into existing services as much as possible.
 Besides exposing already existing Spoofax services in an interactive
```

Comment:
Unrelated to this PR, but this suggests we will expose Spoofax's internals in the REPL?